051-pip-imuxlout-bypalts broken with 'pip_loop.mk: No such file or directory'

```
make[1]: Leaving directory '/big/github/SymbiFlow/prjxray/fuzzers/050-pip-seed'
make -C 051-pip-imuxlout-bypalts clean
make[1]: Entering directory '/big/github/SymbiFlow/prjxray/fuzzers/051-pip-imuxlout-bypalts'
../int_loop.mk:1: pip_loop.mk: No such file or directory
make[1]: *** No rule to make target 'pip_loop.mk'.  Stop.
make[1]: Leaving directory '/big/github/SymbiFlow/prjxray/fuzzers/051-pip-imuxlout-bypalts'
Makefile:42: recipe for target 'clean' failed
make: *** [clean] Error 2
```

--- a/fuzzers/clb.mk
+++ b/fuzzers/clb.mk
@@ -1,3 +1,5 @@
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+
 N ?= 1
 CLB_DBFIXUP ?=
 # Fuzzer that can accept SLICEL data
@@ -15,7 +17,7 @@ ITER ?= 0
 MAX_ITER ?= 10
 FUZDIR = ${PWD}
 
-include ../fuzzer.mk
+include $(SELF_DIR)/fuzzer.mk
 
 database: build/segbits_clbx.db
 

--- a/fuzzers/int_loop.mk
+++ b/fuzzers/int_loop.mk
@@ -1,4 +1,5 @@
-include pip_loop.mk
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+include $(SELF_DIR)/pip_loop.mk
 
 # Specimens from current run must complete, but previous iterations may exist
 database: $(SPECIMENS_OK)

